docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -1190,6 +1190,16 @@ Default stratifications to all observers should include scenario and input draw.
     - Default
     - Default
     - Default
+  * - 13.1
+    - * Update hemoglobin on maternal disorders PAF values to be location specific (rather than using location_id=1)
+      * Resolve discrepancy between RR values in artifact and RR values in get_draws
```

Comment:
Was this already addressed? I see you removed other references to the discrepancy